### ISSUE TITLE
Download results in very big mp4 file

### ISSUE THREAD

**Issue by Sasski** [Franiac/TwitchLeecher#203]
hello,
I ve noticed that after the updates to 1.4.1 and 1.4.2 the downloads result in disproportionate big files. Although the video quality is the same and the length too, around 1h30 min, the file is now instead of 1,50GB, 3GB or even 4GB.
Is this something related to twitchleecher? If not, do tell me if you have any idea how to reverse this.
thanks!!

**Comment by Franiac** [comment]
There is no coherence between video length and file size. It depends not only on the resolution and framerate of the video but also on the bitrate and the used encoder (so it is dependent on the streamer). A 1.5 hour 10080p@60fps with 6000kb/s VOD is logically bigger than a 720p@30fps with 2500kb/s VOD. So results vary not only between qualities but also between streamers due to their different bitrate and so on. Another factor is the transcoding of Twitch themselves which is also not consistend from what I have expierienced.

To sum this up: TL only downloads VOD fragments and merges them together. It does not care about the size or modifies the incoming video or audio stream. That said, TL has absolutely no impact on how big a resulting video file will be.

**Comment by Sasski** [comment]
oh okay, the same streamer retains the same 1.5h 1080@30fps quality in both but he must have changed the live streaming bitrate only. thanks.

**Comment by Franiac** [comment]
Either that or Twitch messed up. Sometimes there are situations where nobody can really tell what's going on...
